src/novel_applications/payment/private_payment.md

```diff
@@ -0,0 +1,167 @@
+
+While blockchains have largely solved the problems of high gas fees and slow execution/finality through the introduction of alternative layer-1 and layer-2 solutions, two critical issues continue to prevent mass adoption:
+
+1. **User Experience (UI/UX)**: Despite technological advancements, many blockchain applications still suffer from poor UI/UX design. The complexity of interacting with blockchain systems remains a significant barrier for non-technical users, often involving confusing wallet interfaces, complicated address systems, sensitive seed phrases and unintuitive transaction processes.
+
+2. **Privacy**: Most blockchain networks still operate on a public ledger system, where all transactions are visible and traceable. The lack of built-in privacy features means that financial transactions lack the confidentiality that users expect from financial systems. This transparency, while beneficial for some use cases, poses significant concerns for personal and business transactions that require discretion.
+
+These issues have prevented blockchains from acquiring the role of leading payment infrastructure and forced users to continue relying on classical centralized infrastructures (e.g., Visa, Mastercard, PayPal, Venmo, Revolut, banks, etc.).
+
+This raises the question: *What about current private decentralized systems?*
+
+While decentralized privacy-preserving protocols like [Zcash](https://z.cash) and [Monero](https://getmonero.org) exist, they face significant limitations. These systems lack stable assets, exclusively using their native tokens (ZEC and XMR), which prevents the use of widely adopted stablecoins like USDC or USDT. This absence of stable, pegged currencies hinders real-world transactions and broad adoption. Additionally, their transaction fees (Zcash: $0.023, Monero: $0.061), while seemingly low, are still too high to compete effectively with traditional payment systems or newer blockchain solutions.
+
+Miden solves these issues.
+
+## Miden: an improved payment infrastructure
+
+Given the limitations of existing blockchains and payments infrastructure, both public and privacy-focused, there is a clear need for a solution that combines privacy, efficiency, and functionality. This is where Miden enters the picture. By building on the foundational work around privacy of projects like Zcash and Monero and around computation of a project like Ethereum, while also incorporating lessons learned from scalable solutions like rollups, Miden presents a promising approach towards building a more ideal payments infrastructure:
```

Comment:
When you have changed the points above, I would re-write this sentence to "... that combines scalability, privacy, and functionality"